lib/node_modules/@stdlib/stats/base/dists/rademacher/mean/docs/repl.txt

```diff
@@ -0,0 +1,20 @@
+
+{{alias}}()
+    Returns the expected value of a Rademacher distribution.
+
+    Parameters
```

Comment:
The "Parameters" section can be removed here, as the function does not take any parameters.